lighthouse-core/gather/gather-runner.js

```diff
@@ -249,9 +249,9 @@ class GatherRunner {
       return driver.endNetworkCollect();
     }).then(networkRecords => {
       GatherRunner.assertPageLoaded(options.url, driver, networkRecords);
-
-      // Network records only given to gatherers if requested by config.
-      config.recordNetwork && (passData.networkRecords = networkRecords);
+      // expose devtoolsLog & networkRecords to gatherers
```

Comment:
there's also a `// Network is always recorded for internal use, even if not saved as artifact.` comment that needs updating